releases/net472/KnownIssues/560016-XBAP Applications targeting .NET 2.0,3.5 select wrong runtime version.md

```diff
@@ -0,0 +1,28 @@
+# [|Known Issue Title|]
+XBAP Applications targeting .NET 2.0/.NET 3.5 select wrong runtime (v4.0 CLR) instead of v2.0 CLR 
+
+## Symptoms
+[|Description of the symtopms|]
+When a XAML Browser Application (XBAP) targeting .NET 2.0/.NET 3.5 is launched, it is loaded in CLR v4 incorrectly instead of being launched under CLR v2 as is expected. Along with the incorrect CLR, the .NET Framework libraries are also loaded from the latest version of .NET (.NET 4.x), and not .NET 3.5. Many XBAP's launch and work normally under CLR v4, but some XBAP's would fail. The failure-mode would be specific to the individual application and cannot be easily generalized. 
```

Comment:
- .NET --> .NET Framework (all references to .NET should be .NET Framework; .NET is a generic family of products that include .NET Framework, .NET Core, and Xamarin)
- possibly instead of loaded/launched --> it is loaded by CLR v4 instead of CLR v2, as expected.
- not .NET 3.5 --> not from .NET Framework 3.5
XBAP's --> XBAPs (2 occurences)
some XBAPs would fail --> some fail
failure-mode would be specific --> failure mode is specific

Reply:
done.